### ISSUE TITLE
Add missing super() quickFix errors for constructors with access modifier

### ISSUE THREAD

**Issue by mjbvz** [microsoft/TypeScript#29565]
**TypeScript Version:**  3.3.0-rc

**Search Terms:**

**Code**

```ts
class Base {}

class Sub extends Base {
    public constructor() {}
}
```

1. Request quick fixes on the `constructor`

**Bug:**
TS Server error:

```
Error: Debug Failure. False expression.
    at getNode (tsserver.js:112889:22)
    at Object.getCodeActions (tsserver.js:112878:27)
    at tsserver.js:109413:121
    at Object.flatMap (tsserver.js:504:25)
    at Object.getFixes (tsserver.js:109413:23)
    at tsserver.js:118617:35
    at Object.flatMap (tsserver.js:504:25)
    at Object.getCodeFixesAtPosition (tsserver.js:118615:23)
    at IOSession.Session.getCodeFixes (tsserver.js:126768:64)
    at Session.handlers.ts.createMapFromTemplate._a.(anonymous function) (tsserver.js:125573:61)
    at tsserver.js:126937:88
    at IOSession.Session.executeWithRequestId (tsserver.js:126928:28)
    at IOSession.Session.executeCommand (tsserver.js:126937:33)
    at IOSession.Session.onMessage (tsserver.js:126959:35)
    at Interface.<anonymous> (tsserver.js:128220:27)
    at Interface.emit (events.js:182:13)
    at Interface._onLine (readline.js:290:10)
    at Interface._normalWrite (readline.js:433:12)
    at Socket.ondata (readline.js:149:10)
    at Socket.emit (events.js:182:13)
    at addChunk (_stream_readable.js:279:12)
    at readableAddChunk (_stream_readable.js:264:11)
    at Socket.Readable.push (_stream_readable.js:219:10)
    at Pipe.onread (net.js:636:20): Error: TypeScript Server Error (3.3.0-dev.20190119)
```

The quick fix works for `constructor` without an access modifier

**Comment by hoangpham95** [comment]
Can I tackle this?

**Comment by mjbvz** [comment]
@RyanCavanaugh Marking as VS Code priority and fix available since there is a PR for it (#36601)

**Comment by DanielRosenwasser** [comment]
Thanks @a-tarasyuk!
